proxy/src/main/java/com/velocitypowered/proxy/protocol/netty/GS4QueryHandler.java

```diff
@@ -142,6 +175,16 @@ private static void writeString(ByteBuf buf, String string) {
         buf.writeByte(0x00);
     }
 
+    private List<QueryResponse.PluginInformation> getRealPluginInformation() {
+        if(pluginInformationList == null) {
```

Comment:
Not sure if safe, needs to be verified

Reply:
If you're unsure, use the double-check pattern.